aws/ecs/cloudformation-identiorca.json

```diff
@@ -3,6 +3,7 @@
   "Description": "Weave-enabled AWS CloudFormation template to create resources required to run tasks on an ECS cluster.  This template uses a larger example app.",
   "Outputs": {
     "AppFrontend": {
+      "Condition": "DoDeployExampleApp",
```

Comment:
Where is this condition initialized? (sorry if it's a stupid question, I am not too familiar with CF)